src/Uno.UWP/Networking/Connectivity/NetworkInformation.cs

```diff
@@ -26,6 +27,12 @@ static NetworkInformation()
 	/// or null if there is no connection profile with a suitable connection.</returns>
 	public static ConnectionProfile GetInternetConnectionProfile() => ConnectionProfile.GetInternetConnectionProfile();
 
+	/// <summary>
+	/// Gets a list of profiles for connections, active or otherwise, on the local machine.
+	/// </summary>
+	/// <returns>An array of ConnectionProfile objects.</returns>
+	public static IReadOnlyList<ConnectionProfile> GetConnectionProfiles() => [ConnectionProfile.GetInternetConnectionProfile()];
```

Comment:
I don't think that would be the correct/ideal implementation.

Reply:
It is not completely correct, but for now it is the extent of what we support and allows people who use `GetConnectionProfiles` to use our implementation (it is needed for port of MAUI Essentials for Prism)